content/sensu-go/5.5/reference/tessen.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Tessen"
+description: "Tessens sends anonymized data about the Sensu instance to Sensu Inc. You can use sensuctl to view and manage the Tessen configuration. Read the reference to configure Tessen."
+weight: 10
+version: "5.5"
+product: "Sensu Go"
+menu: 
+  sensu-go-5.5:
+    parent: reference
+---
+
+- [Configuring Tessen](#configuring-tessen)
+- [Specification](#tessen-specification)
+- [Example](#tessen-example)
+
+## What is Tessen?
+
+Tessen is the Sensu call-home service.
+Included with every Sensu backend, Tessen sends anonymized data about the Sensu instance to Sensu Inc. every six hours, including:
+
+- The Sensu version
+- The number of Sensu agents and backends in the cluster
+- Whether the Sensu instance uses an [enterprise license][4]
+
+All data submissions are logged for complete transparency and transmitted over HTTPS.
```

Comment:
@nikkiki Is this HTTP or HTTPS?

Reply:
https://github.com/sensu/sensu-go/blob/e28ea7c50a0b7eadb70837e3c81438aa06dc7f12/backend/tessend/tessend.go#L21 😄